.github/scripts/update_generation_config.sh

```diff
@@ -15,7 +15,21 @@ set -e
 function get_latest_released_version() {
     local group_id=$1
     local artifact_id=$2
-    latest=$(curl -s "https://search.maven.org/solrsearch/select?q=g:${group_id}+AND+a:${artifact_id}&core=gav&rows=500&wt=json" | jq -r '.response.docs[] | select(.v | test("^[0-9]+(\\.[0-9]+)*$")) | .v' | sort -V | tail -n 1)
+    local latest
+    json_content=$(curl -s "https://search.maven.org/solrsearch/select?q=g:${group_id}+AND+a:${artifact_id}&core=gav&rows=500&wt=json")
+    status=$(jq -r '.responseHeader.status' <<< "${json_content}")
```

Comment:
This relies on the content of the HTTP request for URL. Use curl command's status code.